server/bin/docker/deploy-startup-standalone.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/sh
 
+WORKING="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+. ${WORKING}/deploy-env.sh
```

Comment:
paths should be quoted when used, in case they have spaces
I recommend passing script through shellcheck (shellcheck.net or CLI version)